src/setup.rs

```diff
@@ -13,7 +13,7 @@ pub fn install_targets() {
     let sysroot_cmd = std::str::from_utf8(&sysroot_cmd).unwrap().trim();
 
     let target_files_url = Path::new(
-        "https://raw.githubusercontent.com/LedgerHQ/ledger-nanos-sdk/master/",
+        "https://raw.githubusercontent.com/LedgerHQ/ledger-device-rust-sdk/master/ledger_device_sdk/",
```

Comment:
```suggestion
        "https://raw.githubusercontent.com/LedgerHQ/ledger-device-rust-sdk/cee5644d6c20ff97b13e79a30caca751b7b52ac8/ledger_device_sdk/",
```
To avoid breaking it again in the future?

Reply:
Should be all set now